src/containers/Network/BarChartVersion.tsx

```diff
@@ -68,8 +69,8 @@ const renderLegend = (stableVersion: string | null, t: any) => (
 
 const stableColorCode = (dataLabel: string, stableVersion: string) => {
   if (dataLabel === stableVersion) return GREEN
-  if (dataLabel < stableVersion) return RED
-  return BLUE
+  if (isLaterVersion(dataLabel, stableVersion)) return BLUE
+  return RED
```

Comment:
Looking at this diff, would it make more sense to call the method `isEarlierVersion` and have it return the opposite result? I think that might make the code a little more readable, because right now I have to think about it a little backwards re sorting